troposphere/static/js/models/Instance.js

```diff
@@ -52,12 +52,14 @@ export default Backbone.Model.extend({
       createOnV1Endpoint: function (options, cb) {
         if (!options.name) throw new Error("Missing name");
         if (!options.size_alias) throw new Error("Missing size_alias");
+        if (!options.allocation_source_id) throw new Error("Missing allocation_source_id");
         if (!options.machine_alias) throw new Error("Missing machine_alias");
 
         var providerId = this.get('provider').uuid,
             identityId = this.get('identity').uuid,
             name = options.name,
             size = options.size_alias,
+            allocation_source = options.allocation_source_id,
```

Comment:
remove this variable, and use `options.allocation_source_id` in its place below. Or rename to 'sourceId`. Don't name an id the name of the object, this isn't good for readability.